cves/CVE-2011-3192.yml

```diff
@@ -32,7 +32,11 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+    This vulnerability involves resource exhaustion leading to a denial of service
```

Comment:
**Grammar**
This vulnerability involves `denial of service by resource exhaustion`

Reply:
Updated, good catch!